pkg/drain/evictor/pod.go

```diff
@@ -29,41 +29,41 @@ import (
 const (
 	daemonSetFatal      = "DaemonSet-managed Pods (use --ignore-daemonsets to ignore)"
 	daemonSetWarning    = "ignoring DaemonSet-managed Pods"
-	localStorageFatal   = "Pods with local storage (use --delete-local-data to override)"
+	localStorageFatal   = "Pods with local storage (use --Delete-local-data to override)"
 	localStorageWarning = "deleting Pods with local storage"
 	unmanagedFatal      = "Pods not managed by ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet (use --force to override)"
 	unmanagedWarning    = "deleting Pods not managed by ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet"
 
-	drainPodAnnotation       = "pod.alpha.kubernetes.io/drain"
+	drainPodAnnotation       = "Pod.alpha.kubernetes.io/drain"
```

Comment:
and here, might be others

Reply:
Goland rename variable leaking into some places it shouldn't, my bad!